addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -399,7 +399,7 @@ local function setupTextEntry(pnBase, sName, sID, sRem, nLen)
     RunConsoleCommand(pnConv, sNew)
   end
   pnText.AllowInput = function(pnSelf, chData)
-    return ((pnSelf:GetText():len() >= nLen) and true or false)
+    return (pnSelf:GetText():len() > nLen)
```

Comment:
What was the reason for switching from `>=` to `>`?

Reply:
Hi,

According to the gmod speck: https://wiki.facepunch.com/gmod/DTextEntry:AllowInput

Must return `true` when input must be stopped, so when we pass 17 we must forbid 18, 19, 20 ... and so on and allow string length `<=` to `17` :)